README.md

```diff
@@ -59,9 +62,22 @@ Manually install [Git Large File Storage](https://git-lfs.github.com/), then clo
 git clone https://github.com/mozilla/DeepSpeech
 ```
 
+## Getting the pre-trained model
+
+If you want to use the default English model for performing speech-to-text, you can download it (along with other important inference material) from the [DeepSpeech releases page](https://github.com/mozilla/DeepSpeech/releases). Alternatively, you can run the following command to download and unzip the files in your current directory:
```

Comment:
nit: I don't like "default" here. Maybe phrase it as "If you want to use a pre-trained American English model for performing speech-to-text,"…